docs/source/user_guide/get_started/examples/pw_tutorial.rst

```diff
@@ -564,83 +597,116 @@ Comments are a special set of properties of the calculation, in the sense
 
-  calc.label
+    builder.metadata.options.resources = {'num_machines': 1}
+    builder.metadata.options.max_wallclock_seconds = 1800
 
-will return the string you previously set (empty by default).
-Another important property to set is the description, which instead does not have a
-limitation on the maximum number of characters::
+More options are available, and can be explored by expanding 
+``builder.metadata.options.`` + ``TAB``.
 
-  calc.description = "A much longer description"
 
-And finally, there is the possibility to add comments to any calculation
-(actually, to any node).
-The peculiarity of comments is that they are user dependent
-(like the comments that you can post on facebook pages), so it is best
-suited to calculation exposed on a website, where you want to remember
-the comments of each user.
-To set a comment, you need first to import the django user, and then
-write it with a dedicated method::
 
-  from aiida.backends.djsite.utils import get_automatic_user
-  calc.add_comment("Some comment", user=get_automatic_user())
 
-The comments can be accessed with this function::
+Launching the calculation
+------------------------
 
-  calc.get_comments_tuple()
 
+If we are satisfied with what you created, it is time to attach all the required inputs 
+to the calculation: ::
+  
+    builder.structure = structure
+    builder.kpoints = kpoints
+    builder.parameters = parameters
 
-Execute
--------
-If we are satisfied with what you created, it is time to store everything
-in the database. 
-Note that after storing it, it will not be possible to modify it
-(nor you should: you risk of compromising the integrity of the database)!
 
-Unless you already stored all the inputs beforehand, you will need to store 
-the inputs before being able to store the calculation itself.
-Since this is a very common operation, there is an utility method that will
-automatically store both all the input nodes of ``calc`` and then ``calc`` 
-itself::
+The data nodes do not need to be stored at this point, as AiiDA will store them upon submission.
 
-  calc.store_all()
 
-Once we store the calculation, it is useful to print its PK (principal key, 
-that is its identifier) that is useful in the following to interact with it::
+To execute the calculation, there are two possible ways:
+
+   - run: the calculation gets executed in the shell, locking it until it is finished;
+   - submit: the calculation is handled to the AiiDA daemon, and it will be running in the background
+
+The commands are explained more in detail in the :ref:`working_processes_launching` documentation of AiiDA.
+
+To run your calculation, you can execute: ::
+
+    from aiida.engine import run
+    results = run(builder)
+
+where the ``results`` variable will contain  a dictionary containing all the nodes that were produced as output.
+Alternatively, it is possible to use either ``run_get_node`` or ``run_get_pk`` to retrive more information 
```

Comment:
change the `run_get_node` and `run_get_pk` into `run.get_pk` and `run.get_node`. These are shortcuts such that one doesn't have to do another import `from aiida.engine import run_get_node, run_get_pk`